src/plone/restapi/tests/test_batching.py

```diff
@@ -35,6 +35,10 @@ def setUp(self):
         self.api_session.headers.update({'Accept': 'application/json'})
         self.api_session.auth = (SITE_OWNER_NAME, SITE_OWNER_PASSWORD)
 
+    def tearDown(self):
```

Comment:
@gforcada is there a specific reason why you call "super" after the other calls? Just wondering...

Reply:
Just in case there is another `tearDown` method up in the chain? I will remove them though, if tests are usually not stacked

Reply:
I get why you call "super" in general. I was just wondering why "super" is not the first line in the "tearDown" method.

Reply:
Oh, sorry, I call it last, as it is tearing down: first you clean your local changes and then let the higher up layers clean, just the opposite as in setUp